app/models/concerns/foreman_openscap/hostgroup_extensions.rb

```diff
@@ -16,6 +16,15 @@ def inherited_policies
       end.uniq
     end
 
+    def openscap_proxy
+      return super if ancestry.nil? || self.openscap_proxy_id.present?
+      ::SmartProxy.find_by(:id => inherited_openscap_proxy_id)
+    end
+
+    def inherited_openscap_proxy_id
+      inherited_ancestry_attribute(:openscap_proxy_id)
```

Comment:
Because I 'borrowed' this from [Katello](https://github.com/Katello/katello/blob/master/app/models/katello/concerns/hostgroup_extensions.rb#L72). If there is no Katello, we define it ourselves, if we run with Katello, no need to define it again.